### ISSUE TITLE
Add generator type: Offshore wind

### ISSUE THREAD

**Issue by toddmedema** [toddmedema/electrify#85]
1. TO RESEARCH: Main factors that determine cost
2. ART: Make a variant of wind icon with waves at the base
3. Download historic offshore wind speeds for locations near the coast and merge into existing weather data as new offshore wind speed column, from https://openweathermap.org/history-bulk
4. Only show in generator build list if the location has offshore wind speed
5. Add "Wind, Offshore (Kph)" to weather forecast graph, if location has it

Good inspiration: https://youtu.be/MuwNBtKpjhQ?si=VT0Gq-MxLLsvBnKa

**Comment by toddmedema** [comment]
# Offshore wind: research + implementation plan

Research pass on all five items, with sources, plus a build order. Two assumptions behind the issue have changed since it was written — noted at the bottom.

---

## 1. Cost drivers, and the numbers

**What actually drives the cost**, roughly in order of weight: water depth and distance from shore (foundation, cabling, vessel time); turbine size (450kW at Vindeby to 15MW today); project scale, since the offshore substation, export cable and vessel mobilisation are near-fixed regardless of turbine count; and vessel/supply-chain availability, which is what turned the 2021-24 spike into cancelled US contracts.

### Primary anchor — EIA / Sargent & Lundy, Dec 2023 (basis for AEO2025), 2023$

| | Offshore, fixed-bottom | Onshore, large |
|---|---|---|
| Capital cost | **$3,689/kW** | $1,489/kW |
| Fixed O&M | **$154.00/kW-yr** | $33.06/kW-yr |
| Variable O&M | $0.00/MWh | $0.00/MWh |
| Reference plant | 900 MW / 15 MW turbines | 200 MW / 2.8 MW turbines |
| Lead time | 36 mo (24 development + 12 construction) | — |
| Operating life | 25 years | — |

Offshore's fixed O&M includes $28.8M/yr of seabed leasing that onshore has no equivalent of ($109.8M/yr maintenance on top).

The older EIA vintage we already cite ([Table 8.2](https://www.eia.gov/outlooks/aeo/assumptions/pdf/table_8.2.pdf), AEO2022, 2021$) is far more pessimistic: 400MW plant, 4-year lead, $4,833/kW base with a 1.25 technological-optimism factor on top for $6,041/kW total, $115.16/kW-yr fixed O&M. Real US projects landed across that gap and beyond — Vineyard Wind 1 (806MW) near $4,000/kW, South Fork (132MW) near $15,000/kW.

### The cost curve is a hump, not a decline

This is the thing that makes offshore different from every other row in `GENERATORS`. Solar and onshore wind are monotonic declines and a single `Math.pow` backwards from 2020 works. **Offshore's real cost got worse before it got better** — projects moved into deeper water and further from shore faster than the industry learned.

| Year | Total installed cost | Capacity factor | Source |
|---|---|---|---|
| 2000 | ~EUR 1.5M/MW | — | Voormolen 2016, Europe |
| 2010 | **$5,409/kW** | 38% | IRENA global wtd avg, 2023$ (Voormolen puts Europe at EUR 4.0M/MW) |
| 2015 | >EUR 4.0M/MW | — | Voormolen 2016 |
| 2021 | $3,052/kW | — | IRENA |
| 2022 | $3,478/kW | 42% | IRENA — the supply-chain spike |
| 2023 | **$2,800/kW** | 41% | IRENA global; EIA puts the US at $3,689/kW |
| 2024 | $2,852/kW | — | IRENA |

IRENA's onshore wind over the same window: $2,272/kW to $1,160/kW, CF 27% to 36% — the clean decline we already model. Offshore's global figures are heavily China-weighted and sit below US costs throughout, so the US anchor is the one consistent with every other row in the table.

A monotonic exponential extrapolated back to 1991 would price offshore at ~$15,000/kW. Proposed instead:

```ts
// Offshore wind is the only line in this table whose real costs got worse before they got
// better: projects moved into deeper water and further from shore faster than the industry
// learned, taking European capex from about EUR1.5m/MW in 2000 to EUR4m/MW in 2010
// (Voormolen et al, Energy Policy 88, 2016). Only afterwards did learning win, with IRENA's
// global weighted average falling from $5,409/kW in 2010 to $2,800/kW in 2023.
// So this peaks in 2010 rather than running monotonically backwards the way solar and
// onshore wind do, and is floored below it - Vindeby was tiny, not a bargain.
function offshoreEraMultiple(year: number): number {
  return year <= 2010
    ? Math.max(1.64, 1.82 * Math.pow(2, (year - 2010) / 9))
    : 1.82 * Math.pow(2, (2010 - year) / 15);
}
```

All-in for a 900MW plant: 1995 $5,470/kW · 2010 $5,960/kW · 2020 $4,110/kW · 2023 $3,690/kW · 2040 $2,340/kW. The 2010 figure lands close to IRENA's measured $5,409/kW, which is the check that matters.

### Largest plant built, for `maxPeakW`

| Year | Project | Capacity | Model gives |
|---|---|---|---|
| 1991 | Vindeby, Denmark | 4.95 MW | 5 MW |
| 2000 | Middelgrunden | 40 MW | 30 MW |
| 2002 | Horns Rev 1 | 160 MW | 48 MW |
| 2010 | Thanet | 300 MW | 340 MW |
| 2013 | London Array | 630 MW | **630 MW** |
| 2018 | Walney Extension | 659 MW | 1,030 MW |
| 2019 | Hornsea 1 | 1,218 MW | **1,290 MW** |
| 2022 | Hornsea 2 | 1,386 MW | 1,500 MW (cap) |

Doubling every 3.5 years from 5MW in 1991, capped at 1.5GW. Undershoots 2002-2008, which is fine and arguably useful — that is exactly the window where the era multiplier is at its floor, so cheap offshore stays small. Single-project size genuinely plateaued after Hornsea; Dogger Bank's 3.6GW is three phased 1.2GW farms, not one plant.

---

## 2. The wind resource — measured, not assumed

I pulled real ERA5 hours from the Open-Meteo archive (the same one `fetch-weather.js` already uses) at eight points and compared `wind_speed_10m` against `wind_speed_100m`. 96 hours per point across 1995, 2005, 2015, 2018:

| Point | 10 m | 100 m | Ratio | Implied alpha |
|---|---|---|---|---|
| New York City | 13.6 kph | 24.5 kph | 1.80 | 0.255 |
| Boston | 15.2 | 26.5 | 1.75 | 0.243 |
| London | 13.8 | 23.3 | 1.69 | 0.227 |
| San Francisco | 11.2 | 17.0 | 1.52 | 0.181 |
| NY Bight lease area | 26.1 | 35.2 | **1.35** | 0.131 |
| Vineyard Wind area | 24.6 | 33.8 | **1.37** | 0.137 |
| Humboldt-ish, CA | 26.0 | 32.4 | **1.25** | 0.096 |
| Dogger Bank | 25.6 | 31.6 | **1.24** | 0.092 |

Offshore shear exponents land at 0.09-0.14, squarely on the literature value of 0.11 +/- 0.03 for near-neutral conditions over ocean (Hsu et al. 1994; NREL uses 0.11 for US marine areas).

> ### The existing onshore model turns out to be accidentally correct
>
> `OUTSKIRTS_WIND_MULTIPLIER` gets applied twice — once in `Game.tsx:967` writing `windKph` into the timeline, and again inside `getWindOutputFactor`. Net of the x2, x2, the 10m to 100m gradient and the /5 derate, the chain turns a station reading into **1.750x itself**. ERA5's actual land ratio is 1.80 at New York, 1.75 at Boston, 1.69 at London.
>
> **Not a bug worth chasing — leave it alone.** But do not reuse it offshore.

That is the key modelling point: offshore is not windier *aloft* (shear is far weaker over water), it is windier *at the surface* — 24-30 kph at 10m against 11-15 kph in a city. So offshore needs its own, much smaller conversion, on its own reading, with no outskirts multiplier. There are no outskirts at sea.

### Resulting capacity factors

Game's own power curve (0 below 3 m/s, linear to rated at 14, cut out at 25) over 1,440 real ERA5 hours per point — the 15th of every month across 1985, 1995, 2005, 2015, 2019:

| Lease area | Serves | 10 m | At 150m hub | x 0.85 losses | Real-world |
|---|---|---|---|---|---|
| Dogger Bank | London | 29.7 kph | 62.7% | **53.3%** | ~55% |
| Vineyard Wind | Boston | 27.6 | 59.6% | **50.7%** | ~48% |
| NY Bight | New York | 26.7 | 55.7% | **47.3%** | ~45% |
| Humboldt-ish | San Francisco | 24.4 | 51.2% | **43.5%** | ~45% |
| Off Oahu | Honolulu | 19.9 | 38.2% | **32.5%** | — |
| Pittsburgh (control) | — | 11.4 | 27.6% | — | — |

The 0.85 is wake + electrical + availability losses across a large array — real, documented, and what brings our crude linear power curve back in line with NREL ATB's ~45% for fixed-bottom (a real turbine is cubic between cut-in and rated, so a straight line overstates the middle).

The Pittsburgh control matters: we currently give PIT 21.7%, and the same power curve fed genuine ERA5 100m wind gives 25.4%. Close enough that offshore built this way sits on the same footing as onshore rather than a different scale.

**One quirk to expect:** Honolulu is the only place offshore comes out *worse* than onshore (32.5% vs 49.0%). Trade winds are strong at the surface and barely strengthen with height, and HNL's own 10m reading is unusually high so the 1.75x chain flatters it. Pre-existing over-model, not something offshore introduces — worth a separate look.

---

## 3. The generator entry

```ts
{
  name: "Offshore Wind",
  fuel: "Offshore Wind",
  description: "Steadier and stronger than onshore, at a price",
  available: year > 1991 && hasOffshoreWind(),
    // Vindeby, Denmark, 4.95MW - https://en.wikipedia.org/wiki/Vindeby_Offshore_Wind_Farm
  buildCost: 830000000 + 2.77 * peakW * offshoreEraMultiple(year),
    // $3,689/kW for a 900MW fixed-bottom monopile plant, 2023$ - EIA / Sargent & Lundy,
    // https://www.eia.gov/analysis/studies/powerplants/capitalcost/pdf/capital_cost_AEO2025.pdf
    // Thus the reference plant is $3.32b: 1/4 fixed = $830m, 3/4 variable = $2.77/w
    // Sanity check against real projects: Vineyard Wind 1 (806MW) prices at $3,800/kw here
    // against ~$4,000/kw actual, and South Fork (132MW) at $9,100/kw against ~$15,000/kw -
    // the fixed share is what makes a small farm at sea the bad idea it is in life
  peakW,
  maxPeakW: Math.min(1500000000, 5000000 * Math.pow(2, (year - 1991) / 3.5)),
    // Largest built has roughly doubled every 3.5 years: 4.95MW Vindeby 1991,
    // 630MW London Array 2013, 1.2GW Hornsea 1 2019, and flat since - Dogger Bank's
    // 3.6GW is three phased 1.2GW farms rather than one plant
    // https://en.wikipedia.org/wiki/List_of_offshore_wind_farms
  btuPerWh: 0,
  annualOperatingCost: 0.154 * peakW,
    // $154/kW-yr in 2023 - $122/kW-yr maintenance plus $32/kW-yr of seabed leasing, which
    // onshore has no equivalent of. Nearly 4x onshore wind's $33/kW-yr in the same report
  yearsToBuild: 3 + magnitude / 3,
    // 36 months for the reference plant: 24 development and permitting, 12 construction.
    // The earlier EIA vintage quotes 4 years for a 400MW plant, which is where the
    // magnitude term lands a gigawatt-scale one
  spinMinutes: 1,
  capacityFactor: offshoreWindCapacityFactor,
    // 43-53% measured off ERA5 at real lease areas; IRENA's global weighted average for
    // newly commissioned projects was 41% in 2023, up from 38% in 2010
  lifespanYears: 25,
    // Operating life in the EIA case study, same as onshore wind
},
```

And in `helpers/Energy.tsx`, deliberately not sharing the onshore multiplier:

```ts
// Wind at sea is a different calculation from wind over a city, in both directions. The
// gradient is far weaker - the power law exponent over open water is about 0.11, against the
// 0.34 this file uses over human habitation (Hsu et al, J. Appl. Meteor. 33, 1994; NREL uses
// 0.11 for US marine areas) - but the surface reading it starts from is roughly twice as high,
// which is the whole reason to build out there. There is also no outskirts multiplier: the
// reading is taken at the site rather than at a city weather station miles away.
// Checked against ERA5's own wind_speed_100m at four real lease areas, where the measured
// 10m to 100m ratios came out 1.24 to 1.37 against the 1.29 this produces.
const OFFSHORE_SHEAR_EXPONENT = 0.11;
const OFFSHORE_HUB_M = 150; // IEA 15MW reference turbine, the class EIA prices
const OFFSHORE_ARRAY_LOSSES = 0.85;
  // Wake, electrical and availability losses across a farm of a hundred-plus turbines. Also
  // what brings the linear power curve back to the ~45% NREL reports for fixed-bottom: a real
  // turbine is cubic between cut-in and rated, so a straight line overstates the middle.

export function getOffshoreWindOutputFactor(windKph: number) {
  const turbineWindMS =
    (windKph / KPH_PER_MS) * Math.pow(OFFSHORE_HUB_M / 10, OFFSHORE_SHEAR_EXPONENT);
  return powerCurve(turbineWindMS) * OFFSHORE_ARRAY_LOSSES;
}
```

The curve arithmetic inside `getWindOutputFactor` wants factoring out into a shared `powerCurve(ms)` so both paths cut in at 3, reach rated at 14 and cut out at 25 without two copies drifting apart.

---

## 4. Getting offshore wind into the weather file

**Where the point goes.** `scripts/cities.json` gains an optional offshore coordinate on coastal entries:

```json
{ "id": "NewYork", "city": "New York", "admin": "NY", "lat": 40.7128, "long": -74.006,
  "offshore": { "lat": 40.05, "long": -73.30 } }
```

Hand-curated rather than derived. A "nearest water cell" heuristic has no reliable land/sea signal — Open-Meteo returns elevation 0 for lakes and low ground as readily as ocean — and hand placement puts each point in a real lease area at a sane depth instead of in a harbour. Of the eleven cities with data today, seven get one (SF, LA, HNL, SJU, NewYork, London, Reykjavik) and four do not (PIT, Berlin, Paris, CAMountains) — exactly the split item 4 wants to demonstrate.

**The fetch.** `fetch-weather.js` already batches locations into one request per calendar day, so the offshore point rides along as a second location. Cost doubles for coastal cities — 960 location-days each against the free tier's ~9,500/day — so one run refetches all seven. Inland cities need no refetch.

Fetch `wind_speed_10m` offshore, **not** `wind_speed_100m`: same field, same units, same semantics as the onshore column, so the file stays one thing rather than two heights mixed together, and the height conversion lives in one testable place.

**The format.** `WeatherBinary.tsx` goes to version 2 with a 6-byte row, offshore in the same uint8 half-kph encoding. Header byte 15 is currently reserved — make it a flag for whether the file carries the column, so inland cities keep 5-byte rows and the version 1 files already on disk still decode. The `bytesPerRow` check has to accept both rather than throwing.

A uint8 at half-kph tops out at 127.5 kph and storm hours will exceed that. Harmless: cut-out is 25 m/s ~ 90 kph at hub, so clipped and true readings both produce zero. Worth a comment so nobody later "fixes" it into a wider field.

---

## 5. Determinism — the one thing that must not go wrong

`Weather.tsx` addresses its draws as `dayIndex * DRAWS_PER_FORECAST_DAY + fieldIndex`. Adding offshore as a fourth entry in `FORECAST_FIELDS` takes that constant from 3 to 4, which **shifts the base index for every day of every field**. Every existing seed would produce different temperature and cloud cover, and every stored replay would re-simulate into a different run.

Give it its own stream instead:

```ts
export const RANDOM_STREAM = {
  weather: 1,         // normalAt only -- the per field anomalies
  fuelPrices: 2,      // normalAt only
  economy: 3,         // randomAt only
  weatherShape: 4,    // randomAt only -- which recorded day a forecast borrows its shape from
  weatherOffshore: 5, // normalAt only -- offshore wind's anomaly, on its own stream so that
                      // adding it leaves every existing seed's other three fields untouched
};
```

Drawn as `normalAt(seed, RANDOM_STREAM.weatherOffshore, dayIndex)`, the other three keep their exact indices and every existing seed plays out identically. The shape day is already drawn per day on `weatherShape`, so offshore borrows the same recorded day — right, since the diurnal texture at a city and at its lease area belong to the same weather.

Rest of the surface is cheap: `TickPresentFutureType` gains `windOffshoreKph`, so bump `SAVE_VERSION` and `REPLAY_VERSION` (both version-gated, discard on mismatch, nothing to migrate). `getWeather`'s hour-blending returns an explicit object literal that needs the new field, and `applyClimateForcing` should treat offshore like onshore — scaled spread, no temperature bias.

---

## 6. Build list, chart, art

**Gating (item 4).** Two sources of truth on purpose — the catalogue flag drives the UI before a game loads, the loaded file is the safety net:

- `index.json` gains `offshore: true` per city, written by the fetch script. `CityType` / `LocationType` carry it to the picker.
- `data/Weather.tsx` exports `hasOffshoreWind()`, mirroring the existing `hasFuelPrices()` / `hasEconomy()` — same problem, same shape of answer.
- `available` checks both.

> **Watch this call site.** `technologiesFor` in `CustomGame.tsx:158` builds a synthetic `GameType` with only `date`, `difficulty`, `feePerKgCO2e`, `seed` and `facilities` — **no `location`**. Anything in `available` reading `state.location` crashes the custom game screen. It needs the scenario's location passed in.

**Chart (item 5).** `ChartForecastWeather` already runs two scales. Offshore is a third series sharing the wind scale — same axis, different colour, and worth a dash pattern too, the way `fuelDashArrays` separates the priced fuels. Render only when the location has it; add to the tooltip. One inconsistency to be aware of: the onshore line is drawn at `OUTSKIRTS_WIND_MULTIPLIER x station` while offshore would be the raw reading. Both are honest ("wind where the turbines are") but they are not the same height and the chart does not say so.

**Art (item 2).** Avatars are `/images/{name.toLowerCase()}.svg`, so `name: "Offshore Wind"` wants `public/images/offshore wind.svg` — a space in the filename, which `natural gas.svg` already establishes as fine. Copy `wind.svg`, keep the circular frame and three-blade motif, replace the lower quarter with waves in the sea colour. The existing file is 500x500 with a heavy black keyline; waves need to sit inside that ring rather than crossing it.

---

## 7. Calls I would like your take on

1. **Its own fuel, or shares "Wind"?** *Recommend its own* — the supply-by-fuel chart, dispatch stack and facilities tint can then tell the fleets apart, which is most of the point. Cost is an 8th `fuelColors` entry, and `Theme.tsx` already says seven strains the palette. A mid-cyan against Wind's very dark `#193f79` is the direction; exact hex needs checking against white, Wind, and Natural Gas.
2. **Fixed cost on a 900MW reference ($830m) or 400MW ($369m)?** *Recommend 900MW.* It prices a 100MW farm at ~$10,000/kW, which sounds broken until you notice South Fork's 132MW actually cost ~$15,000/kW. Makes offshore a decision about scale rather than another slider position. If small coastal builds feel dead rather than deliberately bad, AEO2022's 400MW reference halves it.
3. **Floating offshore for the West Coast and Hawaii?** *Recommend not in v1.* The shelf drops fast off CA and HI so real projects there are floating — much pricier, and NREL will not even publish gigawatt-scale floating costs before 2030. Modelling it means a second entry gated on water depth, which means depth data per point. Cost of the shortcut: SF gets cheaper offshore than it would in life.
4. **Refetch all eleven cities or just the coastal seven?** *Recommend coastal only* — with the header flag, inland v1 files keep decoding untouched. ~6,700 location-days, comfortably one run.
5. **"Offshore Wind" or "Wind, Offshore"?** *Recommend "Offshore Wind".* The issue writes the chart series as "Wind, Offshore (Kph)", which reads fine for a weather series but awkwardly as a generator name, and produces `wind, offshore.svg`. A chart series describing wind speed at sea and a plant you build are different nouns.

---

## 8. Build order

Each step independently verifiable, and the long fetch starts first so it runs while the rest is written.

1. **Offshore coordinates and the fetch** — `offshore` on seven `cities.json` entries; teach `fetch-weather.js` to fetch the second point, encode the sixth byte, set the header flag, write `offshore` into `index.json`. Kick off the refetch, it runs for hours.
2. **Decoder, both versions** — v1 and v2, with and without the column. The test file already builds synthetic buffers, so extend those rather than depending on a download.
3. **Forecast the new field** — on `RANDOM_STREAM.weatherOffshore`. **Add a test asserting an existing seed still produces identical temperature, cloud and onshore wind.** That is the regression that would otherwise be found by a player whose replay stopped matching.
4. **Output factor and capacity factor** — factor out the shared `powerCurve`, add the offshore pair. Test cut-in, cut-out, and that a given reading yields more offshore than onshore.
5. **The generator entry** — the `GENERATORS` row, `offshoreEraMultiple`, new fuel in `FuelNameType` / `fuelColors` / `MUST_RUN_FUELS`, and the `Game.tsx` dispatch switch. Test unavailable in 1985, available in 2000, absent where the location has no offshore data.
6. **Wire the timeline** — `windOffshoreKph` into `TickPresentFutureType` and `reforecastWeatherAndPrices`; bump `SAVE_VERSION` and `REPLAY_VERSION`; pass the location into `technologiesFor`.
7. **Chart and art.**
8. **Balance pass** — `npm run sim` against New York or London across a few decades. Watch whether offshore's capacity factor plus its fixed cost make it a real alternative to gas at scale or a trap, and whether a 2010 start can afford it at all at the peak of the era multiplier.

---

## What moved since this issue was written

**The data source.** This issue points at [OpenWeatherMap history-bulk](https://openweathermap.org/history-bulk). We have since replaced the CSVs with the Open-Meteo ERA5 archive. Offshore should come from the same place — ERA5 is a global reanalysis so ocean cells are covered, and an offshore point is just another lat/long in the same batched request. No new vendor, key, or format.

**ERA5 now serves `wind_speed_100m`**, which is what made the shear check in section 2 possible — a way to test the existing wind model against reality instead of guessing at a second fudge factor.

Also worth flagging: the [inspiration video](https://youtu.be/MuwNBtKpjhQ) is about *floating* offshore. Everything costed here is fixed-bottom monopile, which is what exists at scale and what EIA prices.

---

## Sources

**Costs and performance**

- [EIA / Sargent & Lundy, *Capital Cost and Performance Characteristic Estimates for Utility Scale Electric Power Generating Technologies*](https://www.eia.gov/analysis/studies/powerplants/capitalcost/pdf/capital_cost_AEO2025.pdf), Dec 2023 — Case 15 (fixed-bottom offshore, 900MW) and Case 13 (onshore, 200MW). Primary anchor.
- [EIA, *Cost and Performance Characteristics of New Generating Technologies*, AEO2022](https://www.eia.gov/outlooks/aeo/assumptions/pdf/table_8.2.pdf) — the table we already cite.
- [IRENA, *Renewable Power Generation Costs in 2023*](https://www.irena.org/-/media/Files/IRENA/Agency/Publication/2024/Sep/IRENA_Renewable_power_generation_costs_in_2023_executive_summary.pdf) — the 2010 to 2023 series and capacity factors.
- Voormolen, Junginger & van Sark, "Unravelling historical cost developments of offshore wind energy in Europe", *Energy Policy* 88 (2016) 435-444 — 46 European farms, the EUR1.5M to EUR4.0M/MW rise and its depth-and-distance attribution. [Open copy](https://dspace.library.uu.nl/bitstream/handle/1874/329948/1_s2.0_S0301421515301749_main.pdf?sequence=1).
- [NREL Annual Technology Baseline 2024, Offshore Wind](https://atb.nrel.gov/electricity/2024/offshore_wind) — fixed-bottom mature at ~1,008MW plants; floating not costed at GW scale before 2030.
- [EIA, "Cancellations reduce expected U.S. capacity of offshore wind facilities"](https://www.eia.gov/todayinenergy/detail.php?id=62445) — the 2021-24 spike, ~50% LCOE increase, 5.5GW of terminated offtake.

**Wind resource**

- [Open-Meteo Historical Weather API (ERA5)](https://open-meteo.com/en/docs/historical-weather-api) — `wind_speed_10m` and `wind_speed_100m`, 0.25 degree grid, ocean cells covered. The measurements above are pulls from this endpoint.
- Hsu, Meindl & Gilhousen, "Determining the Power-Law Wind-Profile Exponent under Near-Neutral Stability Conditions at Sea", *J. Appl. Meteor. Climatol.* 33:6 (1994) — [alpha = 0.11 +/- 0.03 over ocean](https://journals.ametsoc.org/view/journals/apme/33/6/1520-0450_1994_033_0757_dtplwp_2_0_co_2.xml).
- [NREL Global Offshore Wind GIS Data](https://data.openei.org/submissions/351) — 0.11 used to extrapolate 10m to hub height for US marine areas.
- [List of offshore wind farms](https://en.wikipedia.org/wiki/List_of_offshore_wind_farms) and [Vindeby](https://en.wikipedia.org/wiki/Vindeby_Offshore_Wind_Farm) — the plant-size timeline.

*Measurements: 96 hours per point across four sample days for the shear table; 1,440 hours per point (the 15th of every month across 1985, 1995, 2005, 2015, 2019) for the capacity factors. In-game figures computed by decoding the eleven shipped `.bin` files and running the current `getWindOutputFactor` chain over them.*
